Index: models/Users.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- models/Users.js	(revision 3434702d9ada1654ad873daa76694de9cb8608ef)
+++ models/Users.js	(date 1574670537812)
@@ -1,7 +1,9 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+const Access = mongoose.model('Access');
 
 const UsersSchema = exports.UsersSchema = new Schema({
+  access : Access,
   firstName: {type: String, required: true},
   lastName: {type: String, required: true},
   description: {type: String},
Index: models/Access.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- models/Access.js	(date 1574670537810)
+++ models/Access.js	(date 1574670537810)
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+const Joi = require('joi');
+const Schema = mongoose.Schema;
+
+const Access = exports.Access = new Schema({
+    username: {type: String,
+        required: true,
+        minlength: 5,
+        maxlength: 50},
+    password : {
+        type: String,
+        required: true,
+        minlength: 5,
+        maxlength: 256
+    }
+});
+
+function validateAccess(access) {
+    const obj = {
+        username : Joi.string().min(5).max(50).required(),
+        password: Joi.string().min(5).max(256).required()
+    };
+    return Joi.validate(access, obj);
+}
+
+mongoose.model('Access', Access);
+exports.validate = validateAccess;
\ No newline at end of file
Index: views/user-register.dust
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- views/user-register.dust	(date 1574670335721)
+++ views/user-register.dust	(date 1574670335721)
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{title}</title>
+    <link rel='stylesheet' href='/stylesheets/style.css' />
+</head>
+<body>
+<form action="/register" method="post">
+    <div>
+        <label>First name:
+            <input type="text" name="firstName"/>
+        </label>
+    </div>
+    <div>
+        <label>Last name:
+            <input type="text" name="lastname"/>
+        </label>
+    </div>
+    <div>
+        <label>Description:
+            <input type="text" name="description"/>
+        </label>
+    </div>
+    <div>
+        <label>Choose yor photo:
+            <input type="file" name="image"/>
+        </label>
+    </div>
+    <div>
+        <label>birthday:
+            <input type="date" name="birthday"/>
+        </label>
+    </div>
+    <div>
+        <label>Sex:<br>
+            <input type="radio" name="gender" value="male"> Male<br>
+            <input type="radio" name="gender" value="female"> Female<br>
+            <input type="radio" name="gender" value="other"> Other
+        </label>
+    </div>
+    <div>
+        <label>Email:
+            <input type="email" name="email"/>
+        </label>
+    </div>
+    <div>
+        <label>Username:
+            <input type="text" name="username"/>
+        </label>
+    </div>
+    <div>
+        <label>Password:
+            <input type="password" name="password"/>
+        </label>
+    </div>
+    <div>
+        <input type="submit" value="Register"/>
+    </div>
+</form>
+</body>
+</html>
\ No newline at end of file
